action.yml

```diff
@@ -9,6 +9,9 @@ inputs:
   tag:
     description: 'Tag for swift.org builds. Only specifiy when using official Swift toolchains from swift.org'
     required: false
+  build_arch:
+    description: 'Build architecture (amd64 or arm64). Only specifiy when using official Swift toolchains from swift.org'
```

Comment:
I think that we should default this to `amd64` and require it.

Reply:
nvm

Reply:
Done